### PR TITLE
Done My 4th PR

### PR DESCRIPTION
**Pull request by praveen2896**
adding profile image in respective folder and changes done in team.md

### CODE REVIEW

**Review by Akshay2996 — CHANGES_REQUESTED** [review]
Please see your image. It's stretched due to which the column is also stretched.

**Review by Akshay2996 — APPROVED** [review]
@proful Review this & approve it👍

**Comment by proful** [comment]
@Akshay2996 we need another approval

**Comment by arthurbuhl** [comment]
### Adding profile image in respective folder

@praveen2896: Its seems you have uploaded a 1MB image to the folder. Can you resize the image to 100x100 or 200x200 resolution and upload it again.

### Changes done in team.md

Have you removed these changes? Can you add them again. Thanks!

**Comment by praveen2896** [comment]
@arthurbuhl @Akshay2996 Reduced image resolution , please help to review and approve

File: playground/cli/json-to-pdf-cli/script.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      return;
+    }
+  });
+  //convert the HTML Content/HTML Page to PDF
+  htmlToPdf.convertHTMLFile("output.html", "output.pdf", function (error) {
+    if (error) {
+      console.log("Error Occured ", error);
+      return;
+    }
+    console.log("PDF Succesfully saved in " + process.cwd() + "\\output.pdf");
```
Comment (johnjacobkenny) [inline playground/cli/json-to-pdf-cli/script.js:77]: Using "\\" may not be the ideal way to code the path. I think you should take a look at how to ensure that `\` or `/` is picked correctly based on the OS. Look into the `path` module builtin to nodejs.

Comment (johnjacobkenny) [reply playground/cli/json-to-pdf-cli/script.js:77]: Oh its a comment. Lol never mind.

**Review by johnjacobkenny — APPROVED** [review]

**Pull request merged**